src/analysis/inner_product_space/l2_space.lean

```diff
@@ -387,6 +387,14 @@ begin
   exact subset_span ⟨i, rfl⟩
 end
 
+protected lemma has_sum_inner_mul_inner (b : hilbert_basis ι 𝕜 E) (x y : E) :
+  has_sum (λ i, ⟪x, b i⟫ * ⟪b i, y⟫) ⟪x, y⟫ :=
```

Comment:
Should we have a `tsum` version too?

Reply:
Oh yeah I guess so. I always use `has_sum` so I forget to add `tsum` versions of lemmas :sweat_smile: